### ISSUE TITLE
Documentation indices for Random Samplers incomplete

### ISSUE THREAD

**Issue by asmushetzel** [apache/incubator-mxnet#12518]
## Description
All operators that sample from multiple distributions are not referenced in the "Random sampling" section of ndarray- and symbolic-API (sample_uniform, sample_normal, sample_exponential, ....). As a result, they can only be found by browsing through the full API (where they are documented), means that nobody will be aware off them other than by accident.  They had been correctly referenced some time ago, somehow they got forgotten now. Seems that they also are not part of the random namespace

## Environment info (Required)
MXNet 1.2.1

```

**Comment by asmushetzel** [comment]
Great, looking forward for a fix. Please keep in mind, that these operators should also be part of the "random" namespace. 
Think this is important to fix, people simply won't see what exists so assume that MXNet just has plane nothing for certain tasks. Bad for reputation.

**Comment by sad-** [comment]
@aaronmarkham has the fix for this gone out yet?

**Comment by aaronmarkham** [comment]
No, I don't think it has. I can't see any updates for it. These are also missing on beta.mxnet.io.

**Comment by aaronmarkham** [comment]
@IvyBazan - can you please take a look at this one?

**Comment by IvyBazan** [comment]
It seems that the autosummary sections have specific callouts because of Sphinx limitations with autosummary. I tried a few things like using automodule and tinkering with the autosummary options to automatically include all members of mxnet.symbol.random, but didn't have any luck getting closer to desired functionality.
